File: 27-RemoveElement/27-RemoveElement.cpp
// Last updated: 04/10/2025, 15:55:02
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int count = 0;
        for(int i=0;i<nums.size();i++){
            if(nums[i]!=val){
                nums[count] = nums[i];
                count++;
            }
        }
        return count;
    }
};

File: 27-RemoveElement/27-RemoveElement.java
// Last updated: 04/10/2025, 15:54:58
class Solution {
    public int removeElement(int[] nums, int val) {
        int n = nums.length;
        int k = 0;
        for(int i =0;i<n;i++){
            if(nums[i]!=val){
                nums[k] = nums[i];
                k++;
            }
        }
        return k;
    }
}